doc/releases/changelog-0.23.0.md

```diff
@@ -503,16 +519,15 @@
   - `qml.drawer.CircuitDrawer` is replaced by `qml.drawer.tape_text`.
   - `qml.drawer.CHARSETS` is deleted because we now assume everyone has access to unicode.
   - `Grid` and `qml.drawer.drawable_grid` are removed because the custom data class is replaced
-      by list of sets of operators or measurements.
+    by list of sets of operators or measurements.
   - `RepresentationResolver` is replaced by the `Operator.label` method.
```

Comment:
Of all these changes, I think `Operator.label` is the one that should be elaborated on and is most important to developers.

Reply:
Moved it to the first spot.